src/main/java/com/faforever/api/config/security/WebSecurityConfig.java

```diff
@@ -47,7 +47,9 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
   @Override
   protected void configure(HttpSecurity http) throws Exception {
     // @formatter:off
-    http.headers()
+      http
+      .csrf().disable() // http://stackoverflow.com/a/29917946
```

Comment:
`When should you use CSRF protection? Our recommendation is to use CSRF protection for any request that could be processed by a browser by normal users. If you are only creating a service that is used by non-browser clients, you will likely want to disable CSRF protection.`